controller/script/simulate-proxy/main.go

```diff
@@ -198,7 +274,47 @@ func main() {
 								HttpStatusCode: http.StatusOK,
 							},
 							ResponseLatencies: randomLatencies(count),
-							Ends:              randomEos(count),
+							Ends:              randomGrpcEos(count),
+						},
+					},
+				},
+			},
+		}
+
+		_, err = client.Report(context.Background(), req)
+		if err != nil {
+			log.Fatal(err.Error())
+		}
+
+		// HTTP/2
+		req = &pb.ReportRequest{
+			Process: &pb.Process{
+				ScheduledInstance:  "hello-h2-1mfa0",
+				ScheduledNamespace: "people-h2",
+			},
+			ClientTransports: []*pb.ClientTransport{},
+			ServerTransports: []*pb.ServerTransport{},
+			Proxy:            pb.ReportRequest_INBOUND,
+			Requests: []*pb.RequestScope{
+				&pb.RequestScope{
+					Ctx: &pb.RequestCtx{
+						SourceIp: sourceIp,
+						TargetAddr: &common.TcpAddress{
+							Ip:   targetIp,
+							Port: randomPort(),
+						},
+						Authority: "world.greeting:7778",
+						Method:    &common.HttpMethod{Type: &common.HttpMethod_Registered_{Registered: common.HttpMethod_GET}},
+						Path:      "/World/Greeting",
+					},
+					Count: count,
+					Responses: []*pb.ResponseScope{
+						&pb.ResponseScope{
+							Ctx: &pb.ResponseCtx{
+								HttpStatusCode: http.StatusOK,
```

Comment:
did you mean to use `randomHttpResponseCode()` here? if not, that function is unused...